src/App.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import Calculator from './components/Calculator';
+import QuoteDisplay from './components/QuoteDisplay';
 
 function App() {
   return (
     <div className="App">
       <Calculator />
+      <QuoteDisplay />
```

Comment:
- [ ] well sone on implementing the suggested requirement. but the problem is that you are giving position absolute to `Quote` element and it gives a bad user experience as it is overlapping with the calculator. kindly consider removing lines `67,68` from your `calculator.css` file
your design
<img width="652" alt="Screenshot 2023-06-03 at 12 32 59 AM" src="https://github.com/redmarverick/Math-magicians/assets/96683843/91eb6a62-a780-47db-8c6f-666000b4239a">